tests/lib/commonjs.cjs

```diff
@@ -0,0 +1,60 @@
+/**
+ * @fileoverview Tests for checking that the commonjs entry points are still accessible
+ * @author Mike Reinstein
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const assert = require("assert");
+const tap = require("tap");
+const espree = require("../../dist/espree.cjs");
+
+
+tap.mochaGlobals();
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+describe("commonjs", () => {
+    it("has parse", () => {
+
+        assert.strictEqual(typeof espree.parse, "function");
+
+        const ast = espree.parse("let foo = bar;", {
+            ecmaVersion: 6,
+            comment: true,
+            tokens: true,
+            range: true,
+            loc: true
+        });
```

Comment:
Can you add one test with JSX code?

Reply:
I could, but if we've collectively decided that we want/need to run all of the tests against both esm and cjs, then we should probably just remove `tests/lib/commonjs.cjs` entirely, because at that point it's redundant.